server/lemmy_db/src/comment_view.rs

```diff
@@ -437,7 +455,7 @@ impl<'a> ReplyQueryBuilder<'a> {
     }
 
     query = match self.sort {
-      // SortType::Hot => query.order_by(hot_rank.desc()),
+      // SortType::Hot => query.order_by(hot_rank.desc()), // TODO why is this commented
```

Comment:
git blame says it was this commit: https://github.com/LemmyNet/lemmy/commit/682413613e75618489d262bd033c3101da96abd7#diff-dcc888c431628ad441565ac3bcbf1f37R214